From 815afe0a497e258941964297e54d4849a705dde9 Mon Sep 17 00:00:00 2001
From: Benjamin Lin <benjamin-jw.lin@mediatek.com>
Date: Tue, 14 Jan 2025 14:12:52 +0800
Subject: [PATCH 099/101] mtk: mac80211: add support for EPCS

Define Unsolicited EPCS Priority Access Parameter Update subfield in EHT Capabilities element

Signed-off-by: Benjamin Lin <benjamin-jw.lin@mediatek.com>
---
 include/linux/ieee80211.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/ieee80211.h b/include/linux/ieee80211.h
index f701280..f4fbf3c 100644
--- a/include/linux/ieee80211.h
+++ b/include/linux/ieee80211.h
@@ -3110,6 +3110,7 @@ ieee80211_he_spr_size(const u8 *he_spr_ie)
 #define	IEEE80211_EHT_MAC_CAP0_MAX_MPDU_LEN_11454	        2
 
 #define IEEE80211_EHT_MAC_CAP1_MAX_AMPDU_LEN_MASK		0x01
+#define IEEE80211_EHT_MAC_CAP1_UNSOL_EPCS_PARAM_UPDATE		0x40
 
 /* EHT PHY capabilities as defined in P802.11be_D2.0 section 9.4.2.313.3 */
 #define IEEE80211_EHT_PHY_CAP0_320MHZ_IN_6GHZ			0x02
-- 
2.45.2

